### PR TITLE
Adds the MK22 "Hushpuppy" gun into the game, modified to fire soporific ammo like the ones operatives get.

### PR DESCRIPTION
**Pull request by Militaires**
add: Added a new gun: MK22 Hushpuppy 

[mgs memes.](https://www.youtube.com/watch?v=ItvclICnwgI)
ammo
hush
hush_e

p.s: it's a meme gun so dont expect it to be fair.

`+		L.Sleeping(20)
`

### CODE REVIEW

**Comment by Shapsy** [comment]
>you don't have to manually charge the slide after each shot

**Comment by Militaires** [comment]
@Shapsy im a newbie shit coder how i do dat.

dont worry im making cqc martial arts it'll make up.

**Comment by MMMiracles** [comment]
Bioterror syringes and the mini-syringe gun already accomplishes this idea, no?

**Comment by Militaires** [comment]
@MMMiracles i believe those injects chems that you need to wait for them to take place.

**Comment by MMMiracles** [comment]
Bioterror syringes are instant and non-lethal. 

For the manual loading of each bullet, you might want to just make the gun a subtype of the mosin bolt-action or just looking how it does it if you don't like having a pistol technically being a rifle (which is technically a shotgun,  whew.)

**Comment by Militaires** [comment]
@MMMiracles  barman shotguns are revolvers.

wew

**Comment by Cheridan** [comment]
>sleepypen but ranged

[only screaming]

**Comment by ExcessiveUseOfCobblestone** [comment]
WHAT A THRILL

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 1]
```diff
@@ -74,3 +74,9 @@
 		user << "<span class='notice'>..and falls into view. Whew, that was a close one.</span>"
 		user.unEquip(src)
 
+/obj/item/weapon/gun/projectile/automatic/pistol/mgs
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/projectiles/guns/projectile/pistol.dm]: you should have the name referencing what the object does or it's name [ie pistol/sleepy or pistol/hushpuppy], not the game reference.

**Comment by optimumtact** [comment]
I mean the sprite is nice but *adminonly* weapons are dubious already so I don't see this having much chance of getting in

File: code/modules/projectiles/guns/projectile/pistol.dm (hunk 1 above)
Comment (Militaires) [inline code/modules/projectiles/guns/projectile/pistol.dm]: that'd be more reasonable, yeah, i'll do it.

IM SEARCHING AND IM MELTING INTO YOU

WHAT A FEAR IN MY HEART

**Comment by Militaires** [comment]
@optimumtact it's possible to give this weapon to centcomm special ops that come on the ferry, that'd work, or maybe officials.

or centcomm agents sent to infiltrate and destroy metal gears.

File: code/modules/projectiles/guns/projectile/pistol.dm (hunk 1 above)
Comment (Militaires) [inline code/modules/projectiles/guns/projectile/pistol.dm]: [SNACK EATER](https://www.youtube.com/watch?v=duDLXeC2fLg)

**Comment by kevinz000** [comment]
@optimumtact  you mean like my prs filled with adminonly weapons?

**Comment by Anonmare** [comment]
@Militaires Stop giving me ideas I'll never actually do.

**Comment by Militaires** [comment]
@Anonmare i give my life

not for honor

but for you

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm willing to bet you could put this in a syndiebox with a Krav Maga Implant and not get too much of a fuss considering we put like a spacesuit, bulldog, and some other crazy stuff all in one box, although that would probably be a separate pr.

**Comment by Militaires** [comment]
@ExcessiveUseOfCobblestone  once cqc is finished it wont be a krav maga implant :^)

**Comment by optimumtact** [comment]
@ExcessiveUseOfCobblestone this is literally the sleepy pen but ranged

no

**Comment by Militaires** [comment]
@optimumtact it really is.

**Comment by ExcessiveUseOfCobblestone** [comment]
`if("metaops") // 30 tc
			new /obj/item/clothing/suit/space/hardsuit/syndi(src) // 8 tc
			new /obj/item/weapon/gun/projectile/automatic/shotgun/bulldog/unrestricted(src) // 8 tc
			new /obj/item/weapon/implanter/explosive(src) // 2 tc
			new /obj/item/ammo_box/magazine/m12g/buckshot(src) // 2 tc
			new /obj/item/ammo_box/magazine/m12g/buckshot(src) // 2 tc
			new /obj/item/weapon/grenade/plastic/c4 (src) // 1 tc
			new /obj/item/weapon/grenade/plastic/c4 (src) // 1 tc
			new /obj/item/weapon/card/emag(src) // 6 tc
			return`

and `new /obj/item/weapon/katana(src) // Unique , hard to tell how much tc this is worth. 8 tc?
			new /obj/item/weapon/implanter/adrenalin(src) // 8 tc
			new /obj/item/weapon/throwing_star(src) // ~5 tc for all 6
			new /obj/item/weapon/throwing_star(src)
			new /obj/item/weapon/throwing_star(src)
			new /obj/item/weapon/throwing_star(src)
			new /obj/item/weapon/throwing_star(src)
			new /obj/item/weapon/throwing_star(src)
			new /obj/item/weapon/storage/belt/military/assault(src) // 3 tc
			new /obj/item/weapon/card/id/syndicate(src) // 2 tc
			new /obj/item/device/chameleon(src) // 7 tc`

> but a ranged sleepypen with limited ammo + a snowflake MA wouldn't be ok.

**Comment by Xhuis** [comment]
What is it for? Where is it used? Who gets it? We already have sooo many admin-only things that never get used.

**Comment by Xhuis** [comment]
Um... what are you doing?

**Comment by Militaires** [comment]
@Xhuis hold on a second

oh no i have fucked it up  hold on

**Comment by Militaires** [comment]
tfw you push to the wrong branch and include the gigantic refactor in a simple gun pr kek

**Comment by Militaires** [comment]
@Bawhoppen 

you're a dumb weapon

**Comment by Militaires** [comment]
oh no i fucked the branch shit new pr later

**Pull request closed without merging**